lib-rpc-server/ocamlformat_rpc.ml

```diff
@@ -82,7 +82,8 @@ let run_format conf x =
     (* The formatting functions are ordered in such a way that the ones
        expecting a keyword first (like signatures) are placed before the more
        general ones (like toplevel phrases). Parsing a file as `--impl` with
-       `ocamlformat` processes it as a use file (toplevel phrases) anyway.
+       `ocamlformat` processes it as a use file (toplevel phrases)
+       anyway.
```

Comment:
This is a regression.

Reply:
Did you find what causes this regression? (not important, just wondering)

Reply:
After a few testing, it seems not directly caused by the raw `\n` character because the same happens with `@;<1000 -1000>@\n`.

I have not found a way to print an empty reliably.

Reply:
This should be fixed in a400535

Reply:
Nice! Thanks a lot :)